cli/version/version.go

```diff
@@ -71,9 +78,13 @@ func PrintVersionString(projectPath string, projectPathIsSet bool, showRocksVers
 	}
 
 	if showRocksVersions {
-		if err := printRocksVersion(projectPath); err != nil {
+		if err := printRocksVersion(projectPath, rocksInfo.Versions); err != nil {
 			return fmt.Errorf("%s: %s", rocksVersionsGetError, err)
 		}
+
+		if len(rocksInfo.Duplicates) != 0 {
+			log.Warnf("Duplicate rocks found: %s", strings.Join(rocksInfo.Duplicates, ", "))
```

Comment:
I like warning from `pack/app_dir` more - `log.Warnf("Found multiple versions in rocks manifest: %s", strings.Join(rocksInfo.Duplicates, ", "))`.